mllib-local/src/test/scala/org/apache/spark/ml/linalg/MatricesSuite.scala

```diff
@@ -862,10 +862,10 @@ class MatricesSuite extends SparkMLFunSuite {
     mat.toString(0, 0)
     mat.toString(Int.MinValue, Int.MinValue)
     mat.toString(Int.MaxValue, Int.MaxValue)
-    var lines = mat.toString(6, 50).lines.toArray
+    var lines = mat.toString(6, 50).split('\n')
```

Comment:
Seems like this change is not related to what the PR title says. Just a cleanup?

Reply:
Actually, I added these to fix build failure in the first try failed at https://github.com/apache/spark/pull/30241 . But, it seems that there was something wrong in my local environement at that time. I'll revert this.

Reply:
Oh, I see.